charts/uptrace/values.yaml

```diff
@@ -108,6 +108,11 @@ tolerations: []
 
 affinity: {}
 
+## Provided env variables for uptrace container
+## You can use those variables in uptrace.config
+## Values are same as here https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
+providedEnv: {}
```

Comment:
What do you think about moving this to `uptrace` section so we have `uptrace.env` and `uptrace.config`?

Reply:
sure